requirements/github.in

```diff
@@ -0,0 +1,2 @@
+# As latest version of enmerkar does not support old versions of django<2.2
+git+https://github.com/Zegocover/enmerkar.git@f2ddabaf4da36efa212cd45fb37f0f8bd7335cde#egg=django-babel==0.6.3.dev0
```

Comment:
Are you sure this is the commit you want to use?  This was the last commit in the `django-babel` repo before it was forked, and predates the addition of Django 2.1 and 2.2 support.

Reply:
yeah. because named releases of enmerkar do not support django1.11

Reply:
True, but commit [dbc113798aa4beabdfa2d00e6fef48248eb0f185](https://github.com/Zegocover/enmerkar/tree/dbc113798aa4beabdfa2d00e6fef48248eb0f185) ("Small cleanup based on python-babel#45") seems to include all the fixes for 2.x support just before the commit that dropped 1.11 support ("Rename and adopt").

Reply:
Might have missed that. I will check it out.

Reply:
It seems that commit it working fine. I will test the changes on platform before merging this.